xml/System.Net.Http/HttpClient.xml

```diff
@@ -363,9 +363,36 @@ public class GoodController : ApiController
         <ReturnType>System.Net.IWebProxy</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the global Http proxy.</summary>
+        <value>A proxy used by every call that instantiates a <see cref="T:System.Net.Http.HttpWebRequest" />.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+The <xref:System.Net.Http.HttpClient.DefaultProxy%2A> property gets or sets the global proxy. The <xref:System.Net.Http.HttpClient.DefaultProxy%2A> property determines the default proxy that all <xref:System.Net.Http.HttpClient> instances use if no proxy is set explicitly in the <xref:System.Net.Http.HttpClientHandler> passed through its constructor. 
```

Comment:
Is the %2A here and elsewhere correct? You may have meant
```
<xref:System.Net.Http.HttpClient.DefaultProxy/>
```

Reply:
`%2A` is a star character, not a forward-slash. See [here](https://www.w3schools.com/tags/ref_urlencode.asp). Not sure if it was intended or not, so it might be a good idea for @Jozkee to double check.

In markdown code, the xref elements should not be closed with a forward slash. So the original text is correct.

Reply:
I saw the `%2A` on other properties/methods, I though it was needed when you were not referencing a Type (class/interface/etc.), but it seems is not, therefore I will proceed to remove those `%2A`s. In fact I will remove the entire xref in order to avoid self-reference links.

Reply:
I would still like @mairaw and @rpetrusha to chime in: Is the %2A character allowed in markdown xrefs?